public/app/features/transformers/standardTransformers.ts

```diff
@@ -4,7 +4,7 @@ import { config } from '@grafana/runtime';
 import { filterByValueTransformRegistryItem } from './FilterByValueTransformer/FilterByValueTransformerEditor';
 import { heatmapTransformRegistryItem } from './calculateHeatmap/HeatmapTransformerEditor';
 import { configFromQueryTransformRegistryItem } from './configFromQuery/ConfigFromQueryTransformerEditor';
-import { calculateFieldTransformRegistryItem } from './editors/CalculateFieldTransformerEditor';
+import { calculateFieldTransformRegistryItem } from './editors/CalculateFieldTransformerEditor/CalculateFieldTransformerEditor';
```

Comment:
This is more of a naming nit: Maybe we can leave the dir as `CalculateFieldTransformer`, similar to how other transformer editors who have multiple components are named, so the path would be `'./editors/CalculateFieldTransformer/CalculateFieldTransformerEditor'`